gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -354,15 +302,69 @@ handlers:
     response:
       strict: false
 
+repositories:
+  # Management repository is used to store global configuration such as domains, clients, ...
+  # This is the default configuration using MongoDB (single server)
+  # For more information about MongoDB configuration, please have a look to:
+  # - http://api.mongodb.org/java/current/com/mongodb/MongoClientOptions.html
+  management:
+    type: mongodb
+    mongodb:
+      dbname: ${ds.mongodb.dbname}
+      host: ${ds.mongodb.host}
+      port: ${ds.mongodb.port}
+  #    username:
+  #    password:
+  #    authSource:
+  #    connectionsPerHost: 0
+  #    connectTimeout: 500
+  #    maxWaitTime: 120000
+  #    socketTimeout: 500
+  #    socketKeepAlive: false
+  #    maxConnectionLifeTime: 0
+  #    maxConnectionIdleTime: 0
+  #    serverSelectionTimeout: 0
+  #    description: gravitee.io
+  #    heartbeatFrequency: 10000
+  #    minHeartbeatFrequency: 500
+  #    heartbeatConnectTimeout: 1000
+  #    heartbeatSocketTimeout: 20000
+  #    localThreshold: 15
+  #    minConnectionsPerHost: 0
+  #    sslEnabled: false
+  #    threadsAllowedToBlockForConnectionMultiplier: 5
+  #    cursorFinalizerEnabled: true
+  #    ensureIndexOnStart: true
+  #    cursorMaxTime: 60000
+  #    keystore:
+  #      path:                      # Path to the keystore (when sslEnabled is true, default null)
+  #      type:                      # Type of the keystore, supports jks, pem, pkcs12 (when sslEnabled is true, default null)
+  #      password:                  # KeyStore password (when sslEnabled is true, default null)
+  #      keyPassword:               # Password for recovering keys in the KeyStore (when sslEnabled is true, default null)
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**    keyPassword:               **************************************************************************************  
[This comment was created by Aqua Pipeline]